tools/windows-node-installer/README.md

```diff
@@ -41,10 +41,13 @@ Available Commands:
 
```

Comment:
Why do you need this commit in this PR? https://github.com/openshift/windows-machine-config-operator/pull/134 has already merged.

Reply:
There must have been an issue when rebasing, I'll remove it